src/lifecycle_point_stamped_incr.cpp

```diff
@@ -0,0 +1,955 @@
+            declare_parameter("z_axis_dec", "UNUSED");
+            const std::string z_dec_assignment = get_parameter("z_axis_dec").as_string();
+
+            // const std::string enable_assignment = rosnu::declare_and_get_param<std::string>("enable_control", "UNUSED", *this, "Button assigned to enable control inputs");
+            // const std::string reset_assignment = rosnu::declare_and_get_param<std::string>("reset_enable", "UNUSED", *this, "Button assigned to reset robot position");
+            // const std::string alt_assignment = rosnu::declare_and_get_param<std::string>("alt_enable", "UNUSED", *this, "Button assigned to activate alternative max values");
+            // const std::string x_inc_assignment = rosnu::declare_and_get_param<std::string>("x_axis_inc", "UNUSED", *this, "Button assigned to increase the x-axis value of the robot");
+            // const std::string x_dec_assignment = rosnu::declare_and_get_param<std::string>("x_axis_dec", "UNUSED", *this, "Button assigned to decrease the x-axis value of the robot");
+            // const std::string y_inc_assignment = rosnu::declare_and_get_param<std::string>("y_axis_inc", "UNUSED", *this, "Button assigned to increase the y-axis value of the robot");
+            // const std::string y_dec_assignment = rosnu::declare_and_get_param<std::string>("y_axis_dec", "UNUSED", *this, "Button assigned to decrease the y-axis value of the robot");
+            // const std::string z_inc_assignment = rosnu::declare_and_get_param<std::string>("z_axis_inc", "UNUSED", *this, "Button assigned to increase the z-axis value of the robot");
+            // const std::string z_dec_assignment = rosnu::declare_and_get_param<std::string>("z_axis_dec", "UNUSED", *this, "Button assigned to decrease the z-axis value of the robot");
+            
+            // Additional parameters
+            declare_parameter("x_max", 0.);
+            x_max = get_parameter("x_max").as_double();
+            declare_parameter("y_max", 0.);
+            y_max = get_parameter("y_max").as_double();
+            declare_parameter("z_max", 0.);
+            z_max = get_parameter("z_max").as_double();
+            declare_parameter("alt_x_max", 0.);
+            alt_x_max = get_parameter("alt_x_max").as_double();
+            declare_parameter("alt_y_max", 0.);
+            alt_y_max = get_parameter("alt_y_max").as_double();
+            declare_parameter("alt_z_max", 0.);
+            alt_z_max = get_parameter("alt_z_max").as_double();
+            declare_parameter("x_flip", false);
+            x_flip = get_parameter("x_flip").as_bool();
+            declare_parameter("y_flip", false);
+            y_flip = get_parameter("y_flip").as_bool();
+            declare_parameter("z_flip", false);
+            z_flip = get_parameter("z_flip").as_bool();
+
+            // RCLCPP_INFO(rclcpp::get_logger("lifecycle_point_stamped_incr"), "TESTING PARAMETERS");
```

Comment:
No commented out code

Reply:
Okay I removed all the commented out debugging statements